src/CashFlows.py

```diff
@@ -1027,10 +1027,14 @@ def extendParameters(self, toExtend, t):
               new = np.zeros(t)
               new[0] = float(value)
               toExtend[name] = new
+            elif type(value) is str:
```

Comment:
This type is passed from 12 out of the 14 tests from TEAL tests. The reason they were failing earlier yesterday is because string type data was being passed to extendParameters in CashFlows.py. When that happens, the original condition of "utils.isIntorFloat(value) or (len(value)==1 and utils.isIntorFloat(value[0]))" lets string type alpha/driver to pass without being extended.

Earlier yesterday's code checked the "else" case and extended it regardless of its type. To accommodate, I have added the case of "if type(value) is str -> pass." 

Now it would be great to use "if type(value) == pyomo.blah.blah...." to avoid coincidental extension of non-int-float-str data types being passed as objects but there is no guarantee the type of pyomo expression is always the same depending on what started as a pyomo expression. If the user, chose something other than alpha/driver to make expression variables, we would effectively receive a different expression type.

Reply:
I'm interested to know what those 12 tests are passing in as a string; is this the case where a variable is used as an `alpha` or `driver`? I agree that passing is consistent with the old behavior; I would like to clarify these options in the code so it's easier to know what's happening here. Did the error message tell you what the string was, or just tell you the string didn't work?

Reply:
The error shown was:

TypeError: ufunc 'true_divide' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''

After googling this, it is an error that sent from pandas when you're attempting to use a string for a int/float operation.

Reply:
okay; lots of questions remain for this, but I'm fine with this part working as it did before.